src/Symfony/Component/Serializer/Tests/Normalizer/Features/CallbacksTestTrait.php

```diff
@@ -166,14 +166,7 @@ public function provideNormalizeCallbacks()
             ],
             'Collect a property' => [
                 [
-                    'bar' => function (array $bars) {
-                        $result = '';
-                        foreach ($bars as $bar) {
-                            $result .= $bar->bar;
```

Comment:
are we sure about this change? we don't read `->bar` directly in the new version

Reply:
Oh, you're right! I totally missed that.

Reply:
Reverted this. Sorry.